### ISSUE TITLE
Goerli2 / testnet2's chain ID is incorrect

### ISSUE THREAD

**Issue by Mirko-von-Leipzig** [eqlabs/pathfinder#711]
The chain ID for the new testnet2 is incorrect. `starknet_chainId` returns `SN_GOERLI2` but should return `SN_GOERLI`, the same as for the primary testnet.

**Comment by sergey-melnychuk** [comment]
Current testnet2 chain ID is defined as `0x534e5f474f45524c4932` (different from testnet chain ID `0x534e5f474f45524c49` by extra two digits `32`)

https://github.com/eqlabs/pathfinder/blob/57849ec710453441cb9f032c3a3c87a2a7ac1f8a/crates/pathfinder/src/core.rs#L422

https://github.com/eqlabs/pathfinder/commit/0b158292be8c221a21907eb15f27ece5bc8525c6

**Comment by Mirko-von-Leipzig** [comment]
> Current testnet2 chain ID is defined as `0x534e5f474f45524c4932` (different from testnet chain ID `0x534e5f474f45524c49` by extra two digits `32`)
> 
> https://github.com/eqlabs/pathfinder/blob/57849ec710453441cb9f032c3a3c87a2a7ac1f8a/crates/pathfinder/src/core.rs#L422
> 
> [0b15829](https://github.com/eqlabs/pathfinder/commit/0b158292be8c221a21907eb15f27ece5bc8525c6)

Yes I think @kkovaacs had set it to `SN_GOERLI2` which is the extra `32` you are seeing. This is the byte translation of the string. Or something like that.

**Comment by Mirko-von-Leipzig** [comment]
@sergey-melnychuk Ah I see. The issue is that for whatever reason, those two chain's should have the same ID. The "fix" should be as simple as merging the two testnet arms in [Chain::starknet_chain_id](https://github.com/eqlabs/pathfinder/blob/57849ec710453441cb9f032c3a3c87a2a7ac1f8a/crates/pathfinder/src/core.rs#L415).

So calling `Chain::starknet_chain_id` with either `Testnet` or `Testnet2` should return the same thing = `SN_GOERLI` (in bytes).
